src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * Class that handles loading and saving of tasks for Duke Bot.
+ */
+public class Storage {
+    private FileWriter output;
```

Comment:
Perhaps a more intuitive name could be used, that suggests the attribute is a filewriter